orchagent/fdborch.h

```diff
@@ -62,12 +110,11 @@ class FdbOrch: public Orch, public Subject, public Observer
 
     void updateVlanMember(const VlanMemberUpdate&);
     void updatePortOperState(const PortOperStateUpdate&);
-    bool addFdbEntry(const FdbEntry&, const string&);
-    bool removeFdbEntry(const FdbEntry&);
-    void flushFDBEntries(sai_object_id_t bridge_port_oid,
-                         sai_object_id_t vlan_oid);
+    bool addFdbEntry(const FdbEntry&, const string&, const string&, FdbOrigin origin, const string& remote_ip="", unsigned int vni=0, const string& esi="");
```

Comment:
Why not use FDBData structure here, instead of passing 6 extra arguments?

Reply:
Done.